sql/core/src/test/resources/sql-tests/analyzer-results/ansi/interval.sql.out

```diff
@@ -979,114 +979,96 @@ Project [INTERVAL '30' DAY AS days#x]
 -- !query
 select interval '20 15:40:32.99899999' day to hour
 -- !query analysis
-org.apache.spark.sql.catalyst.parser.ParseException
+org.apache.spark.SparkIllegalArgumentException
 {
-  "errorClass" : "_LEGACY_ERROR_TEMP_0063",
+  "errorClass" : "INVALID_INTERVAL_FORMAT.UNMATCHED_FORMAT_STRING_WITH_NOTICE",
+  "sqlState" : "22006",
   "messageParameters" : {
-    "msg" : "Interval string does not match day-time format of `[+|-]d h`, `INTERVAL [+|-]'[+|-]d h' DAY TO HOUR` when cast to interval day to hour: 20 15:40:32.99899999, set spark.sql.legacy.fromDayTimeString.enabled to true to restore the behavior before Spark 3.0."
-  },
-  "queryContext" : [ {
-    "objectType" : "",
-    "objectName" : "",
-    "startIndex" : 17,
-    "stopIndex" : 50,
-    "fragment" : "'20 15:40:32.99899999' day to hour"
-  } ]
```

Comment:
It is not good that you lost query context here. Could you try to add it back, please.

Reply:
This was generated in the golden file over which unfortunately I do not have control